new_customers.sql

```diff
@@ -0,0 +1,5 @@
+SELECT orders.customer_id, customers.name, customers.email, MIN(orders.created_at) AS first_order_at,
+COUNT(orders.customer_id) AS number_of_orders FROM 
+`analytics-engineers-club.coffee_shop.orders` orders  RIGHT JOIN 
+`analytics-engineers-club.coffee_shop.customers` customers  ON orders.customer_id = customers.id
+GROUP BY orders.customer_id, customers.name, customers.email;
```

Comment:
Hey @lohiers01 ! I think the only thing missing in your query is the ORDER BY and LIMIT at the end. 

This is what I had: "order by first_order_at limit 5" 

Besides that, it works perfectly! I'll approve the PR.

Reply:
Updated PR.